packages/gatsby-plugin-operations/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "compilerOptions": {
```

Comment:
Where did you get all of these settings? Just curious.

My latest strategy for `tsconfig.json`: start with empty, only add settings which are really required.

Reply:
Copy & Paste. I reviewed them a little.